core/src/i18n/en.yaml

```
@@ -634,9 +637,65 @@ deployments:
     error: Error
     running: Running
   title: Deployments
-  tooltips:
-    error: At least one of the Pods has failed
-    running: All the Pods are running correctly
+destination-rules:
+  age: Age
+  description: <0>Destination Rule</0> specifies rules that apply to traffic intended for a service after routing.
+  details:
+    alt-names: Subject Alt Names
+    base-ejection: Base Ejection Time
+    ca-certificates: Ca Certificates
+    client-certificate: Client Certificate
+    client-protocol: Use Client Protocol
+    connect-timeout: Connect Timeout
+    consecutive-5xx: Consecutive 5xx Errors
+    consecutive-failures: Consecutive Local Origin Failures
+    consistent-hash: Consistent Hash
+    credential-name: Credential Name
+    distribution: Distribution
+    enabled: Enabled
+    failover: Failover
+    failover-priority: Failover Priority
+    from: From
+    h2-upgrade: H2 Upgrade Policy
+    http-cookie: Http cookie
+    http-header: Http Header Name
+    http-max: Http2 Max Requests
+    http-pending: Http1 Max Pending Requests
+    http-query-parameter: Http Query Parameter Name
+    http-settings: HTTP Connection Pool Settings
+    idle-timeout: Idle Timeout
+    interval: Interval
+    lb-settings: Locality Lb Settings
```

Is Lb "Load Balancer"? Then I'd suggest 
 ```suggestion
    lb-settings: Locality LB Settings
```